extensions/VisualEditor/wikia/modules/ve/ui/widgets/ve.ui.WikiaVideoOptionWidget.js

```diff
@@ -14,20 +14,20 @@
  * @cfg {number} [size] Media thumbnail size
  */
 ve.ui.WikiaVideoOptionWidget = function VeUiWikiaVideoOptionWidget( config ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this function anonymous by removing its name: 'function() {...}'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ANamedFunctionExpression)